### PR TITLE
Update Scoring for Speech Summarization from NLG-Eval to Huggingface Evaluate

### PR DESCRIPTION
**Pull request by roshansh-cmu**
The NLG-Eval doesn't seem to be updated recently and is incompatible with the latest versions of python, numpy. 

This PR modifies all scoring to rely on the huggingface evaluate package instead. 

Note:

- The ROUGE-L scores are slightly higher than previously obtained using NLG-Eval. However, the NLP community uses evaluate nowadays to compute scores, so it makes sense to retain this. 
- I also updated local/score.sh because the ls command format does not work with certain shell types 
- I removed any checks on nlgeval and bert_score wherever they appear and added the evaluate package to setup.py for installation.

### CODE REVIEW

**Comment by roshansh-cmu** [comment]
Requesting a review from @Jungjee if you have any time ? Thanks !

File: setup.py [hunk 1]
```diff
@@ -87,6 +87,7 @@
         "fairscale",
         "transformers",
         "gtn==0.0.0",
+        "evaluate",
```
Comment (Jungjee) [inline setup.py:90]: can you add the version that worked for you?

**Review by Jungjee — COMMENTED** [review]
LGTM! Just added one minor comment.

File: setup.py (hunk 1 above)
Comment (roshansh-cmu) [reply setup.py:90]: I think the evaluate codebase is updated with transformers, so maybe getting the latest version like transformers is best ?

**Comment by sw005320** [comment]
LGTM!

**Comment by sw005320** [comment]
I also made a label SSUM by considering the growth of this area. This is the first PR/issue having it.

**Pull request merged**